test/controllers/graphql_controller_2_test.rb

```diff
@@ -312,7 +312,7 @@ def setup
     pm = create_project_media project: p
     pm1 = create_project_media project: p, user: u
     create_relationship source_id: pm.id, target_id: pm1.id
-    pm1.archived = true
+    pm1.archived = 1
```

Comment:
Why not use the constants in the test too?

Reply:
Fixed by using constants